### PR TITLE
fix: Uniswap V3 Positions counted double in networth

### LINKED ISSUE #7276: Unsiwap v3 Positions counted double
## Problem Definition

Uniswap v3 LP position NFTs are counted once in the LP token balances, but today I noticed that it also showed up under NFT balances, specifically https://etherscan.io/nft/0xC36442b4a4522E871399CD717aBDD847Ab11FE88/598940. I manually ignored it from the NFT positions, but it would be nice if that'd happen automatically.

### System Description

Operating system: macOS 13.2.1 
Rotki version: 1.13.2

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #7276

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/premium/register-components.ts [hunk 1]
```diff
@@ -134,6 +135,8 @@ export function registerComponents(): void {
   // Version 24 - 1.31
   Vue.component('LpPoolHeader', LpPoolHeader);
   Vue.component('RowAppend', RowAppend);
+  // Version 25 - 1.32
+  Vue.component('UniswapPoolAssetBalance', UniswapPoolAssetBalance);
```
Comment (kelsos) [inline frontend/app/src/premium/register-components.ts:139]: we need to bump the version number somewhere too right?

Comment (lukicenturi) [reply frontend/app/src/premium/register-components.ts:139]: in the premium? yes

Comment (lukicenturi) [reply frontend/app/src/premium/register-components.ts:139]: @kelsos only this line I need to update right? https://github.com/rotki/rotki/blob/b94eefa9b16a002b6f901d53dd15a4b4671fd91b/frontend/app/src/premium/setup-interface.ts#L108

Comment (kelsos) [reply frontend/app/src/premium/register-components.ts:139]: yes yes :+1:

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**